src/screens/Asset/ReceiveModal.js

```diff
@@ -117,24 +118,35 @@ const ReceiveModal = ({
       <ContentWrapper forceInset={{ top: 'never', bottom: 'always' }}>
         <InfoView>
           {!!ensName && (
-            <TextWithCopy textToCopy={ensName} toastText={t('toast.ensNameCopiedToClipboard')} iconColor={colors.link}>
+            <TextWithCopy
+              textToCopy={ensName}
+              toastText={t('toast.ensNameCopiedToClipboard')}
+              iconColor={colors.link}
+              textStyle={{ fontSize: fontSizes.big }}
+            >
               {ensName}
             </TextWithCopy>
           )}
           {ensName ? (
-            <WalletAddress>{address}</WalletAddress>
+            <WalletAddress numberOfLines={1} adjustsFontSizeToFit>
+              {address}
+            </WalletAddress>
           ) : (
             <TextWithCopy
               toastText={t('toast.addressCopiedToClipboard')}
               textToCopy={address}
-              textStyle={{ color: colors.basic030 }}
+              textStyle={{ color: colors.basic030, fontSize: fontSizes.regular }}
```

Comment:
regular should be default, so probably no need to pass it here explicitly.

Reply:
Done.